lib/screens_web/templates/layout/app.html.eex

```diff
@@ -5,9 +5,6 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Screens</title>
-    <%= if @app_id != :dup do %>
```

Comment:
Since the condition is "_not_ DUP", it seems like this stylesheet should now be included always, instead of never.

Reply:
Oof, thank you.